lib/ui/shared/zuri_appbar.dart

```diff
@@ -51,7 +53,7 @@ class ZuriAppBar extends StatelessWidget implements PreferredSizeWidget {
   Widget build(BuildContext context) {
     return AppBar(
       automaticallyImplyLeading: false,
-      elevation: 2,
+      elevation: elevation,
```

Comment:
No sir, this will not be possible. We are using **elevation: 2.0** throughout the app.

Kindly change it back